app/scripts/constants/stream.ts

```diff
@@ -0,0 +1,16 @@
+// contexts
+export const CONTENT_SCRIPT = 'metamask-contentscript';
+export const METAMASK_INPAGE = 'metamask-inpage';
+export const PHISHING_WARNING_PAGE = 'metamask-phishing-warning-page';
+
+// stream channels
+export const METAMASK_PROVIDER = 'metamask-provider';
+export const METAMASK_COOKIE_HANDLER = 'metamask-cookie-handler';
+export const PHISHING_SAFELIST = 'metamask-phishing-safelist';
+export const PHISHING_STREAM = 'phishing';
+
+// TODO:LegacyProvider: Delete
```

Comment:
```suggestion
// For more information about these legacy streams, see here:
// https://github.com/MetaMask/metamask-extension/issues/15491
// TODO:LegacyProvider: Delete
```

maybe we keep the above comment, for more context?